packages/azure/src/__tests__/ConsumptionManagement.test.ts

```diff
@@ -428,7 +428,7 @@ describe('Azure Consumption Management Service', () => {
             accountId: subscriptionId,
             accountName: subscriptionName,
             serviceName: 'Bandwidth',
-            cost: 5,
+            cost: 10,
```

Comment:
Do we know the cause of why these test values needed to be updated? Is it due to the default values added?